README.md

```diff
@@ -2,24 +2,243 @@
+
+
+#### Elevation to Stable status
+Provisional elements of the Standard that have been included in at least two minor releases become eligible for elevation to "**Stable**" status. The elevation process begins with an Issue requesting consideration of the change, identifying the specific elements (APIs and attributes) to be included, and a justification for Stable status. Justification should be commensurate with the magnitude of the request - e.g., elevating an API to "Stable" likely requires more justification than a key-value attribute. While there is no clear formula for justifying elevation, typical considerations might include the breadth of adoption (by implementers, system management vendors, and/or programming libraries) and stability of the elements themselves (i.e., time since last modification).
+
+Note that attributes can be proposed for elevation separate from the APIs that use them. Promotion of one or more attributes independent of any API may, for example, occur when a "Stable" API has expressed a desire to utilize them. In such cases, the proposal should clearly identify the "Stable" APIs making the request and provide some justification for it.
+
+When ready for consideration, the author(s) shall create a corresponding PR (linked to the Issue) in a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) state until ready for submission for review and label the PR with the "Elevate" tag, thereby requesting that the matter be placed on the agenda of the next quarterly meeting (subject to the one month deadline). As with the process for Provisional acceptance, the "**Straw Poll**" mechanism is utilized to provide an initial screening of the proposal. Passing of the straw poll _does not_ confer elevation to Stable status, but serves instead as a means for quickly pushing back proposals that lack significant support. This is intended to both protect the time of ASC members and author(s), and to help guide author(s) wishing to continue pursuit of the proposal.
+
+Upon successfully completing the straw poll, the Co-Chairs shall label the PR as "**Eligible**" to indicate that the proposal is eligible for formal consideration by the ASC. Given the significant commitment conferred by the PMIx community to "Stable" elements, formal approval of elevation requires that the ASC conduct a full presentation on the proposal at two consecutive quarterly meetings. At the conclusion of the first meeting, a formal vote of the ASC shall be held (per the above voting rules) to determine if the proposal merits further consideration. The Co-Chairs shall set a GitHub label indicating the result as either "**First Pass**" or "**Pushed Back**".
```

Comment:
Clarify “First Pass” means “First Vote Passed” - maybe just adjust the label name to make it clear.

Reply:
Done